internal/cli/iam/projects/teams/list.go

```diff
@@ -0,0 +1,77 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package teams
+
+import (
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+const listTemplate = `ID{{range .Results}}
+{{.TeamID}}}{{end}}
```

Comment:
[q] can we have name or is not part of the API?

Reply:
We do not have the team name but only `RoleNames []string `json:"roleNames"``

Reply:
https://docs.opsmanager.mongodb.com/current/reference/api/groups/project-get-teams/#response-body
` "results": [
     {
         "links" : [
           {
             "href": "https://{OPSMANAGER-HOST}:{PORT}/api/public/v1.0/groups/{PROJECT-ID}/teams/{TEAM-ID}",
             "rel": "self"
           }
         ],
         "roleNames": [
           "GROUP_READ_ONLY"
         ],
         "teamId": "{TEAM-ID}"
     }
 ],`